### PR TITLE
[#154240758] Minor fixes in bulk-pdf-bill-generation.sh

### PR DESCRIPTION
**Pull request by keymon**
What?
----

We added this temporary `bulk-pdf-bill-generation.sh` script to get the reports in a dirty temporary way.

But it had a pair of minor bug.

Minor bug fixes:

 - Iterate all orgs page by page. In prod we were missing some orgs
   due pagination.
 - Do not remote the <form>, that is done in the app now.
 - Change directory structure to start with YYYY-MM to make it easier
   to upload new reports to drive.

How to review?
-----------

 * Code review. The script has been used to generate the reports, so you can trust me.

Who?
----

Anyone but @keymon.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -85,15 +71,6 @@ func Main() error {
 
 	var wg sync.WaitGroup
 
-	wg.Add(1)
-	go func() {
-		defer wg.Done()
-		defer logger.Info("stopped event collector")
-		logger.Info("starting event collector")
-		collector.Run(ctx)
```
Comment (chrisfarms) [inline main.go]: I think we should keep this running ;-)

Comment (keymon) [reply main.go]: :man_facepalming:

Comment (keymon) [reply main.go]: fixed, sorry

**Comment by chrisfarms** [comment]
tests failing due to repo rename ... waiting until #10 is merged to rebase

**Comment by keymon** [comment]
@chrisfarms #10 has been merged, rebased.

**Pull request merged**